tests/unit/test_repository.py

```
@@ -19,35 +19,63 @@ def _reset_tables_flag():
     repo_mod._TABLES_ENSURED = False
 
 
+@pytest.fixture(autouse=True)
+def _reset_pool_registry():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_reset_pool_registry`

(ANN202)